impeller/tools/impeller.gni

```diff
@@ -12,13 +12,15 @@ declare_args() {
       flutter_runtime_mode == "debug" || flutter_runtime_mode == "profile"
 
   # Whether the Metal backend is enabled.
-  impeller_enable_metal = is_mac || is_ios
+  impeller_enable_metal = (is_mac || is_ios) && target_os != "fuchsia"
 
   # Whether the OpenGLES backend is enabled.
-  impeller_enable_opengles = is_linux || is_win || is_android
+  impeller_enable_opengles =
+      (is_linux || is_win || is_android) && target_os != "fuchsia"
 
   # Whether the Vulkan backend is enabled.
-  impeller_enable_vulkan = is_linux || is_win || is_android
+  impeller_enable_vulkan =
+      (is_linux || is_win || is_android) && target_os != "fuchsia"
 
```

Comment:
Was this necessary? I don't remember where we landed on the embedder.h issue with fuchsia

Reply:
If I don't do this, there are host artifacts built for linux or mac when building Fuchsia where the target_os is Fuchsia, and enable_impeller ends up being true in places where it really shouldn't be.

I'd be happy to separate this into another patch if it helps.

Reply:
Nah thats fine.